### PR TITLE
CreateRspecIssue: CreateCard

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**
When optional `column-id` parametr is provided, action should create a card for the new issue too.

Usage: https://github.com/pavel-mikula-sonarsource/GitHubActionPlayground/blob/master/.github/workflows/CreateMilestone.yml
Test run: https://github.com/pavel-mikula-sonarsource/GitHubActionPlayground/actions/runs/3410734675/jobs/5674054302
Result: https://github.com/pavel-mikula-sonarsource/GitHubActionPlayground/issues/43 issue is present in the project

### CODE REVIEW

File: CreateRspecIssue/CreateRspecIssue.ts [hunk 1]
```diff
@@ -3,12 +3,13 @@ import { OctokitAction } from "../lib/OctokitAction";
 class CreateRspecIssue extends OctokitAction {
     protected async execute(): Promise<void> {
         this.log("Creating issue");
-        const issue = await this.rest.issues.create(this.addRepo({
+        const issue = (await this.rest.issues.create(this.addRepo({
```
Comment (ilia-kebets-sonarsource) [inline CreateRspecIssue/CreateRspecIssue.ts:6]: You can use [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment):

```javascript
const { data } = await this.rest.issues.create(this.addRepo({
            title: `Update RSPEC before ${this.payload.milestone.title} release`,
            milestone: this.payload.milestone.number,
            labels: ["Type: Tooling"]
}));
this.log(`Created issue #${data.number}: ${data.html_url}`);
this.createCardIssue(data, this.getInputNumber("column-id"));

**Review by ilia-kebets-sonarsource — APPROVED** [review]
See my comment, otherwise LGTM!

File: CreateRspecIssue/CreateRspecIssue.ts (hunk 1 above)
Comment (pavel-mikula-sonarsource) [reply CreateRspecIssue/CreateRspecIssue.ts:6]: I used the `(await ... ).data` so the variable name is `issue`. The `data` is not that clear about what it really carries

Comment (ilia-kebets-sonarsource) [reply CreateRspecIssue/CreateRspecIssue.ts:6]: you can use `{ data: issue }` to assign the `data` prop to the var `issue`.

Comment (pavel-mikula-sonarsource) [reply CreateRspecIssue/CreateRspecIssue.ts:6]: Ok, that sounds good (for the next PR)

**Pull request merged**